server/tinylicious/src/index.ts

```diff
@@ -16,11 +16,11 @@ import { TinyliciousRunnerFactory } from "./runnerFactory";
 // Each TCP connect has a delay to allow it to be reuse after close, and unit test make a lot of connection,
 // which might cause port exhaustion.
 
-// Tinylicious includes end points for ths historian for the client. But even we bundle all the service in
-// a monolithic process, it still uses sockets to communicate with the historian service.  For these call,
-// keep the TCP connection open so that they can be reused
+// Tinylicious includes end points for the historian for the client. But even though we bundle all the
+// services in a monolithic process, it still uses sockets to communicate with the historian service.
+// For these calls, keep the TCP connection open so that they can be reused
 // TODO: Set this globally since the Historian use the global Axios default instance.  Make this encapsulated.
-Axios.defaults.httpAgent = new http.Agent({ keepAlive: true });
+Axios.defaults.httpAgent = new Agent({ keepAlive: true });
```

Comment:
keepAlive: true is the default so you don't need to specify it. No harm though, I assume.